### PR TITLE
8267917: mark hotspot containers tests which ignore external VM flags

### PR DESCRIPTION
**Pull request by iignatev**
Hi all,

could you please review this tiny and trivial patch that adds `@requires vm.flagless` to two container tests that, supposedly, ignore external VM flags?

attn: @mseledts

Cheers,
-- Igor

/cc hotspot-runtime hotspot

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8267917](https://bugs.openjdk.java.net/browse/JDK-8267917): mark hotspot containers tests which ignore external VM flags

### Reviewers
 * [Mikhailo Seledtsov](https://openjdk.java.net/census#mseledtsov) (@mseledts - Committer)
 * [Harold Seigel](https://openjdk.java.net/census#hseigel) (@hseigel - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/4241/head:pull/4241` \
`$ git checkout pull/4241`

Update a local copy of the PR: \
`$ git checkout pull/4241` \
`$ git pull https://git.openjdk.java.net/jdk pull/4241/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 4241`

View PR using the GUI difftool: \
`$ git pr show -t 4241`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/4241.diff">https://git.openjdk.java.net/jdk/pull/4241.diff</a>

</details>

### CODE REVIEW

**Comment by mseledts** [comment]
Looks good to me.

**Review by mseledts — APPROVED** [review]

**Review by mseledts — APPROVED** [review]

**Comment by iignatev** [comment]
Thanks, Misha!

can I get a second Review?

-- Igor

**Comment by hseigel** [comment]
Would this change prevent these tests from being running in many cases, such as  if -Xint or -Xcomp was specified for the test run?

Were tests failing because of the external flags?

**Review by hseigel — APPROVED** [review]
LGTM

**Comment by iignatev** [comment]
Thanks, Harold.

/integrate

**Pull request closed without merging**
